LDK/resources/web/LDK/panel/TabbedReportPanel.js

```diff
@@ -1015,6 +1015,7 @@ Ext4.define('LDK.panel.TabbedReportPanel', {
             var cfg = Ext4.apply({}, filterType);
             cfg.tabbedReportPanel = this;
             cfg.filterContext = this.getFilterContext();
+            cfg.caseInsensitive = this.caseInsensitive;
```

Comment:
Why dont you at least set a default value for "caseInsensitive: false" around line 29, so it's not undefined here? It at  least provides some minimal code documentation that this feature even exists.

Reply:
Yeah I thought about that. It's not necessary logically but good to have it called out.  I'll add it.

Reply:
> > 
> 
> This doesn't change any behavior that hasn't opted in with caseInsensitive: true. If that flag is undefined or false then nothing changes. I'm not seeing any use cases that are broken with this change. Can you please describe what use case you are concerned about?

My apologies - i read that diff wrong in an important way on my phone.  Thanks for making the change in TabbedReportPanel to set a default on caseInsensitive (which might make sense to either include a comment explaining it, or name it "caseInsensitiveSubjects", "preserveCaseForSubjects" or something to make it clear where it operates) but otherwise this seems ok. Again apologies for misreading the changes.

Reply:
Ok added a default setting of the flag and changed the name to caseInsensitiveSubjects.  Added a comment as well